src/interface_py/h2o4gpu/solvers/elastic_net.py

```diff
@@ -1382,22 +1397,26 @@ def __init__(
         #  For l1_ratio = 0 the penalty is an L2 penalty.
         # For l1_ratio = 1 it is an L1 penalty.
         # For 0 < l1_ratio < 1, the penalty is a combination of L1 and L2.
-        alpha_min = alpha_max = l1_ratio
+        #alpha_min = alpha_max = l1_ratio
 
         # Other parameters
-        n_threads = None
-        n_alphas = 1
-        n_lambdas = 1
-        n_folds = 1
-        lambda_max = alpha
-        lambda_min_ratio = 1.0
-        store_full_path = 1
-        alphas = None
-        lambdas = None
+        # n_threads = None
+        # n_alphas = 1
+        # n_lambdas = 1
+        # n_folds = 1
+        # lambda_max = alpha
+        # lambda_min_ratio = 1.0
+        # store_full_path = 1
+        # alphas = None
+        # lambdas = None
```

Comment:
why are these commented out?

Reply:
They will now be passed from the constructor. Will remove.